modules/pam_limits/pam_limits.c

```diff
@@ -886,16 +903,20 @@ parse_config_file(pam_handle_t *pamh, const char *uname, uid_t uid, gid_t gid,
         uid_t min_uid = (uid_t)-1, max_uid = (uid_t)-1;
 
         line = buf;
+        /* Rip off the newline char */
+        tptr = strchr(line,'\n');
+        if (tptr)
+            *tptr = '\0';
+
+        _pam_drop(orig);
+        orig = strdup(line);
+
         /* skip the leading white space */
         while (*line && isspace((unsigned char)*line))
             line++;
```

Comment:
Is it really useful to include the leading white space in `orig`?

Reply:
I'd keep the original line as it is so an administrator knows exactly which line is affected and that PAM did not misread it in any way. Alternatively, it might be sufficient to print the filename and line number in the log.

Reply:
I agree, the filename and line number seem to be a better choice.

Reply:
Okay, I've adjusted the message.